circuits/plonk-5-wires/src/wires.rs

```diff
@@ -0,0 +1,38 @@
+/*****************************************************************************************************************
+
+This source file implements Plonk circuit gate wires primitive.
+
+*****************************************************************************************************************/
+
+use algebra::bytes::{FromBytes, ToBytes};
+use std::io::{Read, Result as IoResult, Write};
+
+pub const COLUMNS: usize = 5;
+pub const WIRES: [usize; COLUMNS] = [0,1,2,3,4];
```

Comment:
It would be nicer (and cleaner when converting to OCaml) if we explicitly spell out the wires, use a `Col` enum like 3-wires plonk had, and implement `Index` for indexing into `WIRES` by `Col`.